### PR TITLE
Added linked list implementation of stack and queue

### PR DESCRIPTION
**Pull request by Ishita-Jena**
## Related Issue

This pull request addresses issue fix: #310 

## Description

I've added the linked list approach of stack and queue in python, java and c++.

## Type of PR

- [ ] Bug fix
- [ ] Feature enhancement
- [Yes] Documentation update
- [ ] Other (specify): _______________

## Screenshots / Videos (if applicable)

[Attach any relevant screenshots or videos demonstrating the changes]

## Checklist
- [Yes] I have performed a self-review of my code.
- [Yes] I have read and followed the Contribution Guidelines.
- [Yes] I have tested the changes thoroughly before submitting this pull request.
- [Yes] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [Yes] I have commented my code, particularly in hard-to-understand areas.

## Additional Context

[Include any additional information or context that might be helpful for reviewers.]

## Resources for Guidance

Here are some resources that may be helpful as you contribute to CodeHarborHub:
- [Code Harbor Hub Community Features](https://www.codeharborhub.live/community/features)
- [Docusaurus Documentation](https://docusaurus.io/docs/create-doc)
- [React.js Documentation](https://legacy.reactjs.org/docs/getting-started.html)
- [Markdown Guide](https://www.markdownguide.org/)
- [MDX Documentation](https://mdxjs.com/docs/)
- [Mermaid Documentation](https://mermaid.js.org/)

**Resources for DSA Features on Our Site:**

- Tabs: [Switch Languages](https://www.codeharborhub.live/community/features#multi-language-support-code-blocks)
- [Line Numbering](https://www.codeharborhub.live/community/features#line-numbering)
- [Math Equations](https://www.codeharborhub.live/community/features#math-equations-with-katex)

### CODE REVIEW

File: dsa/beginner/stacks-and-queue-using-linkedlist.md [hunk 1]
```diff
@@ -0,0 +1,675 @@
...
+- **Syntax Checking:** In programming languages, stacks are used to check if the opening and closing symbols such as the parentheses, braces and brackets are matched properly or not. For example, in HTML, a stack is used to check whether each opening tag has a corresponding closing tag.
+
+## Basic Operations Of A Stack
+
+- **Push** - Inserting a new element in the stack.</li>
+- **Pop** - Removing/deleting the top most element in the stack.</li>
+- **Peek** - Viewing the top element of the stack but not deleting it.</li>
+- **isFull** - Checking if the stack is full.</li>
+- **isEmpty** - Checking if the stack is empty.</li>
+</ul>
```
Comment (Ajay-Dhangar) [inline dsa/beginner/stacks-and-queue-using-linkedlist.md:23]: **Bad Practice**

```md
## Basic Operations Of A Stack

- **Push** - Inserting a new element in the stack.</li>
- **Pop** - Removing/deleting the top most element in the stack.</li>
- **Peek** - Viewing the top element of the stack but not deleting it.</li>
- **isFull** - Checking if the stack is full.</li>
- **isEmpty** - Checking if the stack is empty.</li>
</ul>
```

**Good Practice**

```md
## Basic Operations Of A Stack

- **Push:**  Inserting a new element in the stack.
- **Pop:**  Removing/deleting the top most element in the stack.
- **Peek:**  Viewing the top element of the stack but not deleting it.
- **isFull:**  Checking if the stack is full.
- **isEmpty:**  Checking if the stack is empty.
```

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
update code based on suggestions

**Comment by Ajay-Dhangar** [comment]
@Ishita-Jena could you please update it? Otherwise, I may need to close it.

**Comment by Ishita-Jena** [comment]
I made the necessary changes. Please check if all's good

**Comment by Ishita-Jena** [comment]
Hello, I made the necessary changes. But after doing it, it's not passing the quality test though I haven't touched anything else except the changes requested. Would you please tell me the problem?

**Comment by Ajay-Dhangar** [comment]
> Hello, I made the necessary changes. But after doing it, it's not passing the quality test though I haven't touched anything else except the changes requested. Would you please tell me the problem?

@Ishita-Jena This error isn't caused by your code. It appears due to a duplicate line of code and is intended as a best practice reminder.

**Pull request merged**